lib/appeals/responses/appeals.rb

```diff
@@ -14,7 +14,7 @@ def initialize(body, status)
       private
 
       def json_format_is_valid?(body)
-        JSON::Validator.validate!('lib/appeals/schema/appeals.json', body, strict: true)
```

Comment:
Why are you moving away from strict on the validator? Did you run into an issue that you couldn't get around without it being set to false and if so what was that?

Reply:
Removed my question in favor of this similar but more in depth question ^^

Reply:
I'm good with that then, sorry for missing the comment above on it and going straight into the code for review lol.